monai/bundle/scripts.py

```diff
@@ -142,6 +148,45 @@ def _download_from_github(repo: str, download_path: Path, filename: str, progres
     extractall(filepath=filepath, output_dir=download_path, has_base=True)
 
 
+def _add_ngc_prefix(name: str, prefix: str = "monai_"):
+    if name.startswith(prefix):
+        return name
+    return f"{prefix}{name}"
+
+
+def _remove_ngc_prefix(name: str, prefix: str = "monai_"):
+    if name.startswith(prefix):
+        return name[len(prefix) :]
+    return name
+
+
+def _download_from_ngc(download_path: Path, filename: str, version: str, remove_prefix: Optional[str], progress: bool):
+    # ensure prefix is contained
+    filename = _add_ngc_prefix(filename)
+    url = _get_ngc_bundle_url(model_name=filename, version=version)
+    filepath = download_path / f"{filename}_v{version}.zip"
+    if remove_prefix:
+        filename = _remove_ngc_prefix(filename)
```

Comment:
Hi @yiheng-wang-nv ,

I think we should use below code for this line:
```py
filename = _remove_ngc_prefix(filename, prefix=remove_prefix)
```
Current implementation hard-code to use prefix "monai_" not matter what the `remove_prefix` arg string is...
Could you please help correct it?
CC @wyli @mingxin-zheng 

Thanks.

Reply:
thanks, I include that one line change in this PR https://github.com/Project-MONAI/MONAI/pull/5760

Reply:
Cool, thanks for the quick update.